docgen/src/guides/migration.md

```diff
@@ -26,3 +26,24 @@ var search = instantsearch({
   }
 });
 ```
+
+## The items are not sorted like before in the refinementList / menu / hierarchicalMenu widget
+
+We changed the default sort order of those widgets. This might have impacted your implementation
```

Comment:
What about reconcialating both your comments and make the examples for each widget to use the previous scheme?